model/src/main/scala/cromwell/pipeline/model/wrapper/Name.scala

```diff
@@ -4,16 +4,13 @@ import cats.data.{ NonEmptyChain, Validated }
 import cromwell.pipeline.model.validator.Wrapped
 import play.api.libs.json.Format
 import cats.implicits.catsStdShowForString
+import cromwell.pipeline.model.validator.cromwell.pipeline.model.validator.WrappedFactoryPattern
 
 final class Name private (override val unwrap: String) extends AnyVal with Wrapped[String]
 
-object Name extends Wrapped.Companion {
-  type Type = String
-  type Wrapper = Name
-  type Error = String
-  implicit lazy val nameFormat: Format[Name] = wrapperFormat
-  override protected def create(value: String): Name = new Name(value)
-  override protected def validate(value: String): ValidationResult[String] =
+object Name extends WrappedFactoryPattern[String, String, Name] {
+  override def create(value: String): Name = new Name(value)
```

Comment:
Why are you removing Type, Wrapper, Error here, but leaving it in other microtypes?

Reply:
just forgot while fixing problems with git, it must be removed too, of course, thanks